docker-compose.yml

```diff
@@ -22,10 +22,11 @@ services:
       - webpack_watcher
 
   database:
-    image: postgres:10.4-alpine
+    image: postgres:14.4-alpine
```

Comment:
I'd like to move off of alpine but we aren't changing anything and it isn't used in the deployment, I think it's _fine_ to stay on alpine here. But if we ever upgrade the node dependency in the Dockerfile, I'd switch that over to the `node:##-slim` tag.

Reply:
What's the downside to `alpine`? Just curious.